### PR TITLE
SWP-2832 button in plugin settings to sync categories

### PR DESCRIPTION
**Pull request by pasenor**

### CODE REVIEW

File: Organic/ContentSyncCommand.php [hunk 1]
```diff
@@ -77,8 +77,6 @@ public function __invoke( $args, $opts ) {
             return;
         }
 
-        $this->organic->syncCategories();
```
Comment (sergunich) [inline Organic/ContentSyncCommand.php:80]: Why not to sync catergories periodically?

Comment (pasenor) [reply Organic/ContentSyncCommand.php:80]: They were failing and blocking all content sync. It should be fixed with this PR, but just to be sure I'd like to observe how it works, then after a while return this line.

Comment (pasenor) [reply Organic/ContentSyncCommand.php:80]: On second thought it doesn't seem a good idea, put it back. Still worth keeping an eye on the content sync for some days after the release.

**Review by sergunich — APPROVED** [review]
Looks good!

**Pull request merged**